Create introducing-steve-yegge.md (#5814)
sourcegraph/about: content/blogposts/2022/introducing-steve-yegge.md

The massive training data sets from Sourcegraph’s open source SCIP index will power a whole new generation of ML enhancements to code search, code exploration, and code understanding.

This is it. The party’s starting up again. The worldwide compiler community is a very small place. Everyone knows everyone. We’ll be inviting all the communities for languages, static analysis, build systems, code hosts, and the broader dev tooling ecosystem. SCIP is an interchange format. It’s a way for us to agree to move to the next level. A way to create game-changing tools for developers.

Isn’t that what we all want?

With SCIP, Sourcegraph has stepped up to the challenge. And I’m so grateful that they have invited me along for the ride.

If you want to help, you can of course [come join us at Sourcegraph](https://boards.greenhouse.io/sourcegraph91), right from the comfort of your own home; [we are 100% remote](https://handbook.sourcegraph.com/company-info-and-process/remote). That’s one option.

> But you don’t _have_ to join Sourcegraph to be able to party with us. It’s all open source. Everyone’s welcome (you're welcome in our [Discord community](discord.gg/s2qDtYGnAE), too!). And if you’re into search, or compilers, or ML-powered code intelligence, I think you’ll have an absolute blast.

Reviewer: @steveyegge:

```suggestion
But you don’t _have_ to join Sourcegraph to be able to party with us. [Our code](https://sourcegraph.com/github.com/sourcegraph/sourcegraph) and [our development](https://github.com/sourcegraph/sourcegraph/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc) are public. Everyone’s welcome (you're welcome in our [Discord community](discord.gg/s2qDtYGnAE), too!). And if you’re into search, or compilers, or ML-powered code intelligence, I think you’ll have an absolute blast.
```

> But you don’t _have_ to join Sourcegraph to be able to party with us. [Our code](https://sourcegraph.com/github.com/sourcegraph/sourcegraph) and [our development](https://github.com/sourcegraph/sourcegraph/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc) are public. Everyone’s welcome (you're welcome in our [Discord community](discord.gg/s2qDtYGnAE), too!). And if you’re into search, or compilers, or ML-powered code intelligence, I think you’ll have an absolute blast.

Reviewer 2: ```suggestion
But you don’t _have_ to join Sourcegraph to be able to party with us. [Our code](https://sourcegraph.com/github.com/sourcegraph/sourcegraph) and [our development](https://github.com/sourcegraph/sourcegraph/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc) are public. Everyone’s welcome (you're welcome in our [Discord community](https://discord.gg/s2qDtYGnAE), too!). And if you’re into search, or compilers, or ML-powered code intelligence, I think you’ll have an absolute blast.
```
